css/components/_textfield-icon.scss

```diff
@@ -0,0 +1,6 @@
+.textfield-icon {
```

Comment:
Could we relocate this in a file inside the modals subdirectory? I.e. css > modals > _password-required.scss